Mathlib/Analysis/NormedSpace/PiLp.lean

```diff
@@ -89,23 +89,19 @@ namespace PiLp
 
 variable (p : ℝ≥0∞) (𝕜 𝕜' : Type*) {ι : Type*} (α : ι → Type*) (β : ι → Type*)
 
-/-- Canonical bijection between `PiLp p α` and the original Pi type. We introduce it to be able
-to compare the `L^p` and `L^∞` distances through it. -/
-abbrev equiv : PiLp p α ≃ ∀ i : ι, α i := WithLp.equiv _ _
-#align pi_Lp.equiv PiLp.equiv
-
 /-! Note that the unapplied versions of these lemmas are deliberately omitted, as they break
 the use of the type synonym. -/
 
 @[simp]
-theorem equiv_apply (x : PiLp p α) (i : ι) : PiLp.equiv p α x i = x i :=
+theorem _root_.WithLp.equiv_pi_apply (x : PiLp p α) (i : ι) : WithLp.equiv p _ x i = x i :=
```

Comment:
Should this and the things below it go in the `WithLp` file? Probably not, but just a thought.

Reply:
I think it makes sense to leave here, to ensure we don't write lemmas for types that aren't intended to be used with `WithLp` in the first place. These lemmas are only interesting because we add further structure to `WithLp` on pi types in the rest of the file.